Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/OverlaySheetService.java

```diff
@@ -583,13 +585,11 @@ public void addToList(Bundle rideRequestBundle) {
     }
 
     public boolean removeCardById(String id){
-        if (sheetArrayList!=null){
```

Comment:
null check is removed as part of warnings resolution? "sheetArrayList can not be null"

Reply:
It cannot be null because whenever we are defining at that time only we are initializing it.

`private final ArrayList<SheetModel> sheetArrayList = new ArrayList<>();`